metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -24,6 +25,11 @@
 logger = logging.getLogger(__name__)
 
 
+telemetry_enabled = (
```

Comment:
Why is this guy in here?

Reply:
Because in case we had file sink this was causing telemetry client id to fetched. In case telemetry is disabled it should not be doing that.

For unit tests this was doing 3 retries and failing. Adding this and passing the environment variable reduced time for unit tests by 30-60 secs.